### PR TITLE
Allow navigation through homework questions by showing all breadcrumbs

### PR DESCRIPTION
**Pull request by pandafulmanda**
# Homework
screen shot 2015-04-23 at 2 50 34 pm
screen shot 2015-04-23 at 2 50 49 pm
screen shot 2015-04-23 at 6 53 50 pm

# Not Homework
screen shot 2015-04-23 at 2 50 26 pm
screen shot 2015-04-23 at 2 49 49 pm

Using star now but can easily switch for something else.
- [x] tests
  - [x] breadcrumbs show for all steps of homework
  - [x] breadcrumbs are clickable
  - [x] end breadcrumb also clickable and shown

- [x] breadcrumbs are numbered

### CODE REVIEW

**Comment by pandafulmanda** [comment]
@philschatz in mockup, the breadcrumbs include the one for the final turn panel as well...should I add that as well?

**Comment by philschatz** [comment]
Ah, yes I think that `^` thingy needs to be in this

**Comment by pandafulmanda** [comment]
oh okay, that changes things

**Comment by pandafulmanda** [comment]
What about the question mark thing?

**Comment by philschatz** [comment]
I think the question mark thing might be for another PR unless there's already info in the Task JSON. The info would be a step `type="pending"` I believe. @kevinburleigh75 is that in `tutor-server#master`?

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,6 @@
 /.tmp
 /dist
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+*.sublime-project
```
Comment (philschatz) [inline .gitignore]: Should this be in `~/.gitignore`? (probably same with `~/.DS_Store` now that I think about it)

**Comment by pandafulmanda** [comment]
type pending should be very easy to add...maybe ill do that in a stub and style it before this gets merged.

File: style/task-step.less [hunk 2]
```diff
@@ -5,6 +5,25 @@
 @answer-hover-color: #e6f9fc;
 @answer-select-color: #def4f7; // darker than hover
 
+// WHOA. This is sneaky. And hilarious. Am I allowed?!
+// http://caniuse.com/#search=counter
```
Comment (philschatz) [inline style/task-step.less]: I'd prefer you use CSS counters rather than generate them in the JS ; )

**Pull request merged**